frontend/src/services/ApiService.ts

```diff
@@ -24,21 +24,27 @@ export const transformAxiosError = (e: Error): ApiError => {
     if (e.response) {
       const statusCode = e.response.status
       if (statusCode === StatusCodes.TOO_MANY_REQUESTS) {
-        return new HttpError('Please try again later.', statusCode)
+        return new HttpError('Error [429]: Please try again later.', statusCode)
       }
       if (typeof e.response.data === 'string') {
-        return new HttpError(e.response.data, statusCode)
+        return new HttpError(`Error [001]: ${e.response.data}`, statusCode)
       }
       if (e.response.data?.message) {
-        return new HttpError(e.response.data.message, statusCode)
+        return new HttpError(
+          `Error [002]: ${e.response.data.message}`,
+          statusCode,
+        )
       }
       if (e.response.statusText) {
-        return new HttpError(e.response.statusText, statusCode)
+        return new HttpError(
+          `Error [003]: ${e.response.statusText}`,
+          statusCode,
+        )
       }
-      return new HttpError(`Http ${statusCode} error`, statusCode)
+      return new HttpError(`Error [${statusCode}]`, statusCode)
```

Comment:
To be consistent with the other labels, how about Error `[004]: ${statusCode} error`?

Reply:
okie shifted everything down!